### ISSUE TITLE
"Failed to load extension from .../build. Could not load javascript 'static/js/0.chunk.js' for content script.

### ISSUE THREAD

**Issue by xush6528** [xpl/crx-hotreload#19]
I get a pop up error message box from chrome on reloading.

This is my `manifest.json`, https://gist.github.com/xush6528/b2721f54f34fc740e4e2facdd44cb800

 I wonder what other requirements this solution has.

**Comment by xush6528** [comment]
My hypothesis is that the building has not finished yet, the extension reloading is fired, thus resulting chrome to load broken files.

**Comment by xpl** [comment]
Yeah, could be the case. I guess a white-list of watched files could help, so that the extension would reload only when a specific file is updated. A larger timeout could also help.
